Libplanet/Action/ActionEvaluator.cs

```diff
@@ -184,6 +190,7 @@ internal static IImmutableSet<Address> GetUpdatedAddresses(Transaction<T> tx)
         /// Executes <see cref="IAction"/>s in <paramref name="actions"/>.  All other evaluation
         /// calls resolve to this method.
         /// </summary>
+        /// <param name="genesisHash"> A <see cref="BlockHash"/> value of the genesis block </param>
```

Comment:
Our docstring lint is required to line end with a period.
```suggestion
        /// <param name="genesisHash"> A <see cref="BlockHash"/> value of the genesis block. </param>
```